plugins/PrivacyManager/vue/src/ManageGdpr/ManageGdpr.vue

```diff
@@ -525,6 +529,20 @@ export default defineComponent({
         '</a>',
       );
     },
+    siteSettingsText(): string {
+      return translate(
+        'PrivacyManager_PleaseEnableVisitorLogsProfilesSites',
+        `<a href="${this.linkTo('globalSettings', 'SitesManager')}">`,
+        '</a>',
+      );
+    },
+    siteSettingsTextSingle(): string {
+      return translate(
+        'PrivacyManager_PleaseEnableVisitorLogsProfiles',
+        `<a href="${this.linkTo('globalSettings', 'SitesManager')}">`,
+        '</a>',
+      );
+    },
```

Comment:
Am I wrong or do the translation(s) not contain any placeholders where such html tags could be inserted?